tests/integration/kubernetes/k8s-security-context.bats

```diff
@@ -25,7 +32,6 @@ setup() {
 	kubectl wait --for=condition=Ready --timeout=$timeout pod "$pod_name"
 
 	# Check user
-	cmd="ps --user 1000 -f"
 	process="tail -f /dev/null"
 	kubectl exec $pod_name -- sh -c $cmd | grep "$process"
```

Comment:
Hi @danmihai1 ,

One little adjust would be nice: so here pass `$exec_command`, i.e., `kubectl exec $pod_name -- $exec_command | grep "$process"` to ensure it's will always be exactly the command set in the policy settings.

Reply:
That's a great point - thanks @wainersm! I missed that before, so my preference would be to clean up all instances of this bug in a separate PR. I will fix this using the reminder from https://github.com/kata-containers/kata-containers/issues/10014.